lib/utility/strings.rb

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+module Utility
+  class Strings
```

Comment:
```suggestion
module Utility
  class String
```

Reply:
I understand the naming is coming from `Utility::Errors` but we should rename it to `Utility::Error` in a separate PR :D

Reply:
Done 👍